apps/site/lib/site_web/templates/trip_plan/_fare_calculator.html.eex

```diff
@@ -79,3 +79,16 @@
   <% end %>
 
 </div>
+<div class="m-trip-plan-farecalc__notes-block">
+  <span class="m-trip-plan-farecalc__title">Reduced fares:</span> Some riders are eligible for <%= link "reduced fares", to: fare_path(@conn, :show, "reduced") %>. Military personnel and children 11 and under ride for free.
+</div>
+
+<%= if transfer_note != nil do %>
+  <div class="m-trip-plan-farecalc__notes-block">
+    <span class="m-trip-plan-farecalc__title">Transfers:</span> This trip suggestion includes one or more transfers. <%= link "Learn more about transfers", to: fare_path(@conn, :show, "transfers") %>.
+  </div>
+<% end %>
+
+<div>
+  <%= link "See fares overview", to: fare_path(@conn, :show, "") %>
```

Comment:
Sorry to have missed this before, but let's add the `.c-call-to-action` class to this link to get the nice underline!

Reply:
Sure! I thought the underline in Zeroheight was just to indicate it was a hyperlink :sweat_smile: Changing this.